util/Order.h

```diff
@@ -613,6 +614,7 @@ class FO_COMMON_API ShipDesignOrder : public Order {
     void ExecuteImpl() const override;
 
     int m_design_id = INVALID_OBJECT_ID;
+    boost::uuids::uuid m_uuid;
```

Comment:
I've added it to the description.  It is part of make ShipDesignOrder copy the design UUID.  It is there so that when new designs are created by a client AI/human the UUID is copied along with the other ship design data.